openach/templates/boards/common/_pagenav.html

```diff
@@ -21,7 +22,7 @@
                 </li>
             {% else %}
                 <li>
-                    <a href="?page={{ page }}">{{ page }}</a>
+                    <a href="{% relative_url page 'page' request.GET.urlencode %}">{{ page }}</a>
```

Comment:
Instead of passing the request in, I think you want to set `takes_context=True` on the simple_tag definition:  https://docs.djangoproject.com/en/3.1/howto/custom-template-tags/